projeto_guiado_v2_williane.py

```diff
@@ -0,0 +1,187 @@
+                    else:
+                        break
+                except ValueError as e:
+                    print(e)
+        else:
+            nota_recuperacao = None
+            print("Então essa aluna será reprovada por média.")
+    else:
+        status = "Reprovada"
+        nota_recuperacao = None
+    
+    # Adicionando a aluna ao dataset
+    chave = f"{nome} {sobrenome}"
+    dataset[chave] = {
+        "Turma": turma,
+        "Notas": notas_aluna,
+        "Presença": presenca_aluna,
+        "Nota de Participação": participacao,
+        "Status": status,
+        "Média Final": round(media, 2)  # Arredonda a média final para duas casas decimais
+    }
+    
+    # Exibindo o status de aprovação da aluna
+    print(f"\nStatus de aprovação da aluna '{chave}': {status}")
+    print(f"Média final: {media:.2f}")
+    if status == "Recuperação":
+        print(f"Nota de recuperação: {nota_recuperacao:.2f}")
+    
+    print(f"\nAluna '{chave}' adicionada com sucesso!")
+
+def consultar_lista_alunas():
+    if not dataset:
+        print("\nNão há alunas cadastradas.")
+    else:
+        print("\nLista de alunas cadastradas:")
+        for chave in dataset.keys():
+            print(chave)
+
+def consultar_faltas_aluna():
+    nome_aluna = input("\nDigite o nome completo da aluna para consultar as faltas: ")
+    if nome_aluna in dataset:
+        faltas = dataset[nome_aluna]["Presença"].count(False)
+        print(f"\nA aluna '{nome_aluna}' possui {faltas} falta(s).")
+    else:
+        print(f"\nAluna '{nome_aluna}' não encontrada.")
+
+def consultar_notas_aluna():
+    nome_aluna = input("\nDigite o nome completo da aluna para consultar as notas: ")
+    if nome_aluna in dataset:
+        notas = dataset[nome_aluna]["Notas"]
+        print(f"\nNotas da aluna '{nome_aluna}': {notas}")
+    else:
+        print(f"\nAluna '{nome_aluna}' não encontrada.")
+
+def consultar_status_aprovacao():
+    nome_aluna = input("\nDigite o nome completo da aluna para consultar o status de aprovação: ")
+    if nome_aluna in dataset:
+        notas = dataset[nome_aluna]["Notas"]
+        presenca = dataset[nome_aluna]["Presença"]
+        participacao = dataset[nome_aluna]["Nota de Participação"]
+        
+        if len(notas) == 0:
+            print("\nNão há notas registradas para esta aluna.")
+            return
+        
+        media = sum(notas) / len(notas)
+        percent_presenca = presenca.count(True) / len(presenca) * 100
+        
+        if participacao >= 6 and percent_presenca >= 80 and media >= 6:
+            status = "Aprovada"
+        elif 3 <= media < 6:
+            status = "Recuperação"
+        else:
+            status = "Reprovada"
+        
+        print(f"\nStatus de aprovação da aluna '{nome_aluna}': {status}")
+        print(f"Média final: {media:.2f}")
+        if status == "Recuperação":
+            print(f"Nota de recuperação necessária para aprovação: {media:.2f}")
```

Comment:
Como na função incluir_nova_aluna foi adicionado um elemento chamado "Status", podemos só retornar o valor que está nele.

**Sugestão:**
```suggestion
    if nome_aluna in dataset:
        status = dataset[nome_aluna]["Status"]
        
        print(f"\nStatus de aprovação da aluna '{nome_aluna}': {status}")
        print(f"Média final: {media:.2f}")
        if status == "Recuperação":
            print(f"Nota de recuperação necessária para aprovação: {media:.2f}")
```